providers/errorReportingProvider.ts

```diff
@@ -0,0 +1,18 @@
+import { ApplicationContract } from '@ioc:Adonis/Core/Application'
```

Comment:
Can we name the files in kebab-case (error-reporting-provider.ts)?  It makes it easier to follow on case-sensitive OSes.